WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
Concensus is the absence of leadership. In this case, it was an oversight that needed to be done. 

Imagine if everyone had to take up every bug fix or minor change with you. Oh wait, everyone but me doesn't have to.